config/locales/en.yml

```diff
@@ -188,6 +188,12 @@ en:
 
       <h2 class="govuk-heading-m">If you didn’t expect to see this page</h2>
       <p class="govuk-body">You may be seeing this page because you’ve turned off cookies in your browser. You’ll need to <a class="govuk-link" href="https://ico.org.uk/your-data-matters/online/cookies/" rel="external" target="_blank">turn cookies on</a> before you can use this service.</p>
+  address_api:
+    error_400: "Bad Request"
+    error_401: "Unauthorized"
+    error_404: "Not Found"
+    error_405: "Method Not Allowed"
+    error_500: "Internal Server Error"
```

Comment:
I don't expect the class that uses the address lookup class to use this text, so I'm not sure this is necessary in its current form.

From the perspective of someone consuming this class, we'd want the address lookup class to raise an error, or have a method like `.succeeded?` or something like that, rather than returning text.

Reply:
They were only being used in the tests so they've been removed. I expect that when the consumer of AddressLookup is implemented, a decision will be made about whether these entries are needed.